config/services.yml

```diff
@@ -1023,7 +1023,6 @@ https://cmr.uat.earthdata.nasa.gov:
           STAGING_PATH: public/harmony/netcdf-to-zarr
     umm_s:
       - S1237980031-EEDTEST # Just NetCDF-to-Zarr
```

Comment:
We should have only 1 UMM-S in the end. Since DAS owns this service should we ask PODAAC to remove S1241248230-POCLOUD and add any of their associations to S1237980031-EEDTEST?

Reply:
I think that's what I'm expecting to happen. I'm guessing they have a ticket for `services.yml` clean-up, too. (I did the really quick check, but didn't see one linked via dependencies to your work this PI, though)